modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/VersionsContent.tsx

```diff
@@ -113,12 +116,21 @@ const VersionsContent = ({
 					))}
 				</ul>
 			)}
+			{viewVersionsURL && (
```

Comment:
Should we include also this markup in the `loading` if/else block?
It really does not make a difference right now because we can barely see the loading indicator, but I wonder what will happen in slower environments, we could end up viewing something like this?:
![image](https://user-images.githubusercontent.com/19485114/191943170-9158d122-649e-4f46-aa05-a0aab4b50ad4.png)

Reply:
You are right. I'm going to move it inside the loading if/else